src/client/components/atoms/itemList/index.tsx

```diff
@@ -0,0 +1,29 @@
+import { Portfolio } from '../../../utils/Type';
+import getAssetUrl from '../../../utils/getAssetUrl';
+import Button from '../button';
+import './style.scss';
+
+type ItemListProps = {
+  item: Portfolio;
+};
+
+const ItemList = ({ item }: ItemListProps) => {
+  console.log('si', item);
+  return (
+    <div className="itemList">
+      {item.overview.avatarURL ? (
+        <img className="itemList__image " src={item.overview.avatarURL} />
+      ) : (
+        <img className="itemList__image " src={getAssetUrl('placeholder.png')} />
+      )}
+      <div className="itemList__dataContainer">

```

Comment:
wrong use of BEM pattern - we don't use camelCase in classNames

the element className should be something like `item-list__data-container` instead